pages/linux/latexmk.md

```diff
@@ -1,19 +0,0 @@
-# latexmk
-
-> Automated LaTeX document generation.
-
-- Compile a LaTeX file into a PDF file:
-
-`latexmk -pdf {{filename.tex}}`
-
-- Force `latexmk` to generate a document even if there are errors:
-
-`latexmk -f {{filename.tex}}`
-
-- Clean up the current directory from all files generated by `latexmk` except DVI, PostScript and PDF:
```

Comment:
This description is a bit verbose, but maybe a compromise with the shorter "Clean up temporary tex and output files" could be found. How about "Clean up all temporary files in the current directory"?

Reply:
Umm .. this page is removed actually. Look at the common page. Since it had `-c {{source.tex}}` and `-C`, simply keeping `-c` was kinda pointless.

Reply:
Oh, I didn't notice the capitalization difference. That explains the "and output files" in the page being kept -- that confused me. So, with that cleared up, I'd still say `-c` is more useful than `-C`. It's more common to want to clean up the folder (tex processing produces lots of intermediate files) than to remove everything altogether, including the final outputs.

(Btw, I commented here because github doesn't allow commenting on unchanged context lines.)